### DISCUSSION TITLE
Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by ibrartalab** [community/community#126735] [GitHub Education] [answered]
### Hi everyone,

Please some one tell me why my github student developer pack not working on Educative courses. I want to access those courses that are freely available for github students. If anyone know how to get this, then please answer my question.

**Comment by minhhieu-2003** [comment]
I don't log in  github copilot  but i  have verified the email address hieu221a020029@st.vhu.edu.vn on your GitHub account. That academic domain is associated with the school Van Hien University.

**Reply by ibrartalab** [reply, marked as answer]
I didn't get what are you saying. what is this about.

**Comment by minhhieu-2003** [comment]
2024-06-01 09:27:25.931 [info] [auth] Invalid copilot token: missing token: 403 
2024-06-01 09:27:25.949 [error] [auth] Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as minhhieu-2003."

**Reply by ibrartalab** [reply]
Tell me what is this about.

**Comment by minhhieu-2003** [comment]
Sorry I'm wrong
